package.json

```diff
@@ -48,11 +49,11 @@
     "exports": false
   },
   "scripts": {
-    "prepare": "yarn run lint && yarn test && yarn run compile",
-    "compile": "babel -d dist src",
+    "prepare": "concurrently -n \"lint,compile,test\" -c \"blue,yellow,green\" --kill-others-on-fail \"yarn lint\" \"yarn compile\" \"yarn run test\"",
```

Comment:
Instead of escaping these, can they not just be using single quotes – or are double quotes needed for this?

Reply:
I'm not sure, I'll test it out 😸

Reply:
Works with single quotes 😸  I also used the shortened argument names where possible in my latest code push.

Reply:
👍  💯